test/e2e/performanceprofile/functests/utils/cgroup/v2/v2.go

```diff
@@ -0,0 +1,37 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/config"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
+)
+
+type Manager struct {
+}
+
+func (m *Manager) GetConfig(ctx context.Context, c *kubernetes.Clientset, pod *corev1.Pod, containerName string) (*config.Config, error) {
+	var cmd []string
+	cfg := &config.Config{}
```

Comment:
GetConfig here supports getting here only cpuset.cpus, cpu.max, but there are other interface files that equally important and we do need them in future like 
cpuset.cpu.partition, cpuset.cpus.effective, cpuset.mems (this is specifically required for memory manager to know which numa interface the pod has been assigned).